.changeset/salty-cycles-join.md

```diff
@@ -0,0 +1,8 @@
+---
+'@clerk/chrome-extension': minor
+'@clerk/clerk-js': minor
+'@clerk/clerk-expo': minor
+'@clerk/nextjs': minor
+---
+
+Add missing French locales to support new Clerk's Billing feature
```

Comment:
Changesets will automatically handle updating that packages that rely on the localizations, so we can remove this extra changeset here.